src/vllm_tgis_adapter/__main__.py

```diff
@@ -69,3 +308,5 @@ async def _force_log() -> None:
             disable_log_stats=engine_args.disable_log_stats,
         ),
     )
+
```

Comment:
And this is also broken: `grpc_server` is an async generator, while `asyncio.run` expects a coroutine